```
#!/bin/bash -eo pipefail
grep ^otelcontribcol-all-sys Makefile|fmt -w 1|tail -n +2|circleci tests split|xargs make
GOOS=darwin  GOARCH=amd64 make otelcontribcol
make[1]: Entering directory '/home/circleci/project'
GO111MODULE=on CGO_ENABLED=0 go build -trimpath -o ./bin/otelcontribcol_darwin_amd64 \
	-ldflags "-X github.com/open-telemetry/opentelemetry-collector-contrib/internal/version.GitHash=313cc4ef8 -X github.com/open-telemetry/opentelemetry-collector-contrib/internal/version.Version=v0.31.0-3-g313cc4ef8 -X go.opentelemetry.io/collector/internal/version.BuildType=release" ./cmd/otelcontribcol
exporter/newrelicexporter/factory.go:21:2: missing go.sum entry for module providing package github.com/newrelic/newrelic-telemetry-sdk-go/telemetry (imported by github.com/open-telemetry/opentelemetry-collector-contrib/exporter/newrelicexporter); to add:
	go get github.com/open-telemetry/opentelemetry-collector-contrib/exporter/newrelicexporter@v0.0.0-00010101000000-000000000000
make[1]: *** [Makefile:201: otelcontribcol] Error 1
make[1]: Leaving directory '/home/circleci/project'
make: *** [Makefile:215: otelcontribcol-darwin_amd64] Error 2

```

--- a/exporter/newrelicexporter/transformer_test.go
+++ b/exporter/newrelicexporter/transformer_test.go
@@ -26,6 +26,7 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/consumer/consumererror"
 	"go.opentelemetry.io/collector/model/pdata"
 	"go.opentelemetry.io/collector/translator/internaldata"
 	"go.uber.org/zap"
@@ -628,6 +629,7 @@ func TestTransformSum(t *testing.T) {
 				"unit":        "1",
 				"description": "description",
 			},
+			ForceIntervalValid: true,
 		},
 	}
 	expectedGauge := []telemetry.Metric{
@@ -801,7 +803,9 @@ func TestUnsupportedMetricTypes(t *testing.T) {
 		dp.SetBucketCounts([]uint64{1, 1, 0, 0})
 		h.SetAggregationTemporality(pdata.AggregationTemporalityDelta)
 
-		t.Run("DoubleHistogram", func(t *testing.T) { testTransformMetricWithError(t, m, &errUnsupportedMetricType{}) })
+		t.Run("DoubleHistogram", func(t *testing.T) {
+			testTransformMetricWithError(t, m, consumererror.Permanent(&errUnsupportedMetricType{}))
+		})
 	}
 }
 

